onnxruntime/test/optimizer/qdq_test_utils.cc

```diff
@@ -98,5 +98,38 @@ GetQDQTestCaseFn BuildQDQConcatTestCase(const std::vector<std::vector<int64_t>>&
   };
 }
 
+#ifdef USE_NNAPI
```

Comment:
This is a valid test case for CPU EP also, can change name to reflect scales and zps are different